.github/renovate.json

```diff
@@ -4,16 +4,12 @@
   "packageRules": [
     {
       "matchManagers": ["github-actions"],
-      "matchUpdateTypes": ["minor", "patch"],
-      "automerge": true,
       "pinDigests": true,
-      "schedule": ["every weekend after 4am"]
```

Comment:
@tomleb @mallardduck do you see the benefit of expanding this to the global settings to decrease noise? E.g. GH actions are bumped at most once weekly/fortnightly.

Reply:
I think for GHA updates doing it once a fortnight is probably good enough. And will help teams not feel as overloaded as weekly.